ThomasShelby.java

```diff
@@ -31,7 +33,7 @@ private static void markTask(Task[] taskManager, String[] cmdSplit) {
                 + " " + taskManager[whichTask].getDescription());
     }
 
-    private static void unmarkTask(Task[] taskManager, String[] cmdSplit) {
+    private static void unmarkTask(String[] cmdSplit) {
         int whichTask;
         whichTask = Integer.parseInt(cmdSplit[1]) - 1; // shift back index
```

Comment:
I see that you have repeated some code here. You could consider abstracting this into a method for reusability.